src/main/java/teammates/common/datatransfer/questions/FeedbackMcqQuestionDetails.java

```diff
@@ -166,7 +166,7 @@ public String getQuestionWithExistingResponseSubmissionFormHtml(boolean sessionI
                             Slots.RESPONSE_INDEX, Integer.toString(responseIdx),
                             Slots.DISABLED, sessionIsOpen ? "" : "disabled",
                             Slots.CHECKED,
-                                    existingMcqResponse.getAnswerString().equals(choices.get(i)) ? "checked" : "",
+                            existingMcqResponse.getAnswerString().equals(choices.get(i)) ? "checked" : "",
```

Comment:
The change actually decrease the readability :( 
Take a look at the method details and you will know that this is a mapping.